2023/challenge.html

```diff
@@ -192,23 +184,21 @@ <h3 class="title"><span style="color: #dd0a37;">Example Pipeline</span></h3>
 										</p>
 										<p>
 											<ol>
-												<li>The provided CSV files and SQL schema are loaded into a PostgreSQL relational database.</li>
-												<li>RMLMapper executed the provided mappings by accessing the PostgreSQL relational database to construct a knowledge graph in N-Triples as RDF format.</li>
+												<li>The provided CSV files and SQL schema are loaded into a MySQL relational database.</li>
+												<li>Ontop in materalization mode executed the provided mappings by accessing the MySQL relational database to construct a knowledge graph in N-Triples as RDF format.</li>
```

Comment:
do we need to add this? I would remove it (we don't need to say how we have generated the outputs)